docs/source/Howtos/Beginner-Tutorial/Part1/Beginner-Tutorial-Building-Quickstart.md

```diff
@@ -29,137 +28,136 @@ Will give you a list of all commands available to you. Use
 
     create box
 
-This created a new 'box' (of the default object type) in your inventory. Use the command `inventory` (or `i`) to see it. Now, 'box' is a rather short name, let's rename it and tack on a few aliases.
+This creates a new 'box' (of the default object type) in your inventory. Use the command `inventory` (or `i`) to see it. Now, 'box' is a rather short name, so let's rename it and tack on a few aliases:
 
     name box = very large box;box;very;crate
 
-```{warning} MUD clients and semi-colon: 
-Some traditional MUD clients use the semi-colon `;` to separate client inputs. If so, the above line will give an error. You need to change your client to use another command-separator or to put it in 'verbatim' mode. If you still have trouble, use the Evennia web client instead.
+```{warning} MUD Clients and Semi-Colons: 
+Some traditional MUD clients use the semi-colon `;` to separate client inputs. If so, the above line will give an error and you'll need to change your client to use another command-separator or put it in 'verbatim' mode. If you still have trouble, use the Evennia web client instead.
 ```
 
+We have now renamed the box as _very large box_ &mdash; and this is what we will see when looking at it. However, we will also recognize it by any of the other names we have offered as arguments to the name command above (i.e., _crate_ or simply _box_ as before). We also could have given these aliases directly after the name in the initial `create` object command. This is true for all creation commands &mdash; you can always provide a list of `;`-separated aliases to the name of your new object. In our example, if you had not wanted to change the box object's name itself, but to add aliases only, you could use the `alias` command.
 
-We now renamed the box to _very large box_ (and this is what we will see when looking at it), but we will also recognize it by any of the other names we give - like _crate_ or simply _box_ as before. We could have given these aliases directly after the name in the `create` command. This is true for all creation commands - you can always tag on a list of `;`-separated aliases to the name of your new object. If you had wanted to not change the name itself, but to only add aliases, you could have used the `alias` command.
-
-We are currently carrying the box. Let's drop it (there is also a shortcut to create and drop in
-one go by using the `/drop` switch, for example `create/drop box`).
+At this point in the building tutorial, our Character is currently carrying the box. Let's drop it:
 
     drop box
 
-Hey presto - there it is on the ground, in all its normality.
+Hey presto - there it is on the ground, in all its normality. Tthere is also a shortcut to both create and drop an object in one go by using the `/drop` switch immediateliy after the create command (e.g, `create/drop box`). 
```

Comment:
The `immediately after the create command` is superfluous and potentially confusing, since it suggests there may be a time where a `/switch` is _not_ added immediately after the main command name... which is not correct: The switch is a part of the command's normal syntax and always appears 'directly' after the command name. So this should be removed.

Reply:
righto. fixed punctuation that I missed as well.